### DISCUSSION TITLE
Help for Angular 9

### DISCUSSION THREAD

**Discussion by jrmessias** [stevermeister/ngx-cookie-service#270] [Q&A]
Hello

Sorry, I don't know if this is the best place for this, but I need help. Can you help me?

I'm trying to use this library in Angular 9, but the following error occurs:

`peer @angular/common@"^12.0.0" from ngx-cookie-service@12.0.0`

Which version should I use for angular 9?

This is a legacy system, I'm trying to install npm packages for further upgrade.

Thanks

**Comment by pavankjadda** [comment]
You can use version 12.0.3 See compatibility matrix here
https://github.com/stevermeister/ngx-cookie-service#supported-versions

**Reply by jrmessias** [reply]
Using version 12.0.3 the reported error occurs.

`peer @angular/common@"^12.0.0" from ngx-cookie-service@12.0.3`

Thanks

**Reply by pavankjadda** [reply]
Where do you see the error? When doing `npm i` or anywhere else? If that does not help, use version 3.x.x
```
npm i ngx-cookie-service@3.0.0
```
